### PR TITLE
Китаев - Лабораторная работа #1

### PR DESCRIPTION
**Pull request by PavelKitaev**

### CODE REVIEW

**Comment by PavelKitaev** [comment]
@maslichko @Draonsnom

**Comment by maslichko** [comment]
> @maslichko @Dragnsnom

looks good

File: modules/complex-number/test/test_kitaev_pavel_complex_number.cpp [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  double b_im = 28.0;
+
+  // Act
+  ComplexNumber a(a_re, a_im);
+  ComplexNumber b(b_re, b_im);
+  b = a - b;
+
+  // Assert
+  EXPECT_EQ(a_re - b_re, b.getRe());
+  EXPECT_EQ(a_im - b_im, b.getIm());
```
Comment (SHVlader) [inline modules/complex-number/test/test_kitaev_pavel_complex_number.cpp:55]: The data type "double" wants to be checked in EXPECT_DOUBLE_EQ
Please correct all test

**Comment by SHVlader** [comment]
Don't forget about approved
Pull Request -> File chenged -> Review changes -> Approve -> Submit review

**Review by maslichko — APPROVED** [review]

**Review by Dragnsnom — APPROVED** [review]

**Pull request merged**